dds_web/__init__.py

```diff
@@ -164,6 +163,13 @@ def create_app(testing=False, database_uri=None):
         # Set-up the schedulers
         dds_web.utils.scheduler_wrapper()
 
+        from dds_web.api.user import ENCRYPTION_KEY_CHAR_LENGTH
+
+        if len(app.config.get("SECRET_KEY")) != ENCRYPTION_KEY_CHAR_LENGTH:
+            from dds_web.api.errors import KeyLengthError
+
+            raise KeyLengthError(ENCRYPTION_KEY_CHAR_LENGTH)
+
```

Comment:
Should it be equal or at least the number of characters? This checks equal, but the `KeyLengthError` says "at least". Change the KeyLengthError message?

Reply:
This is intentionally written as it is. We don't accept it if it is any shorter than the required length, but we also don't accept it if it is longer than the required length, because a longer key is not fully used by the selected algorithm. It could give a false sense of security with a longer key. Perhaps we can change the error message? For example, remove "at least" from the error message?